internal/cmd/schema-registry/command_schema_create.go

```diff
@@ -56,6 +57,9 @@ func (c *command) newSchemaCreateCommand() *cobra.Command {
 	cmd.Flags().String("subject", "", SubjectUsage)
 	pcmd.AddSchemaTypeFlag(cmd)
 	cmd.Flags().String("references", "", "The path to the references file.")
+	cmd.Flags().String("metadata", "", "The path to metadata file.")
+	cmd.Flags().String("rule-set", "", "The path to schema rule set file.")
+	cmd.Flags().Bool("normalize", false, "Whether to register the normalized schema.")
```

Comment:
Do you mean the following?
```suggestion
	cmd.Flags().Bool("normalize", false, "Whether to normalize the registered schema.")
```

Reply:
Also, can you define what "normalize" means?

Reply:
`normalize` is a query param , if set to true, SR will modify the schema string to have ordered fields.  https://github.com/confluentinc/schema-registry/blob/master/core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/SubjectVersionsResource.java#L376

Reply:
Does that mean it will sort/alphabetize the fields?

Reply:
Yes

Reply:
This might be more clear than "normalize":
```suggestion
	cmd.Flags().Bool("sort", false, "Sort schema fields in alphabetical order.")
```

Reply:
This might be inconsistent with the API query param `normalize`, is that ok?

Reply:
Is it too late to change the API parameter? If yes, it's ultimately up to your team, whether you'd like to prioritize consistency or usability.

Reply:
How about a compromise:
```suggestion
	cmd.Flags().Bool("normalize", false, "Sort schema fields in alphabetical order.")
```